### DISCUSSION TITLE
Ny funktion

### DISCUSSION THREAD

**Discussion by kjeldson** [scaarup/aula#34] [Ideas]

**Comment by scaarup** [comment]
@kjeldson det er som om der ikke længere er nogen tekst på dit indlæg her.

**Comment by Jogge** [comment]
>Når børnene kommer i 7 kl så bruger de ikke ugeplan mere men derimod min uddannelse i aula
en ide kunne være at få den med i din løsning :-)

Havde @kjeldson skrevet.

**Reply by scaarup** [reply]
Ah. Mærkeligt, at det er væk her så...
Nå men ugeplaner fra Min Uddannelse, er allerede understøttet.
